src/AMP.php

```diff
@@ -64,7 +66,7 @@ class AMP
         'Lullabot\AMP\Pass\FacebookNonIframeTransformPass',
         'Lullabot\AMP\Pass\TwitterTransformPass',
         'Lullabot\AMP\Pass\StandardScanPass',
-        'Lullabot\AMP\Pass\StandardFixPass',
+        'Lullabot\AMP\Pass\SugarStandardFixPass',
```

Comment:
No, upstream PR's won't include sugar prefixed files. I'm trying to keep our changes in separate `Sugar` classes to make maintenance easier while I wait for PRs to get merged upstream. The upstream PR I submitted for this changes `StandardFixPass`